src/applications/travel-pay/containers/TravelPayStatusApp.jsx

```diff
@@ -48,9 +49,18 @@ export default function App({ children }) {
 
   if (travelClaims.length > 0 && statusesToFilterBy.length === 0) {
     // Sets initial status filters after travelClaims load
-    const initialStatusFilters = [
+    const claimStatusesPresent = [
       ...new Set(travelClaims.map(claim => claim.claimStatus)),
     ];
+    const initialStatusFilters = [];
+
+    // Status order based on sorting of CLAIMS_STATUSES
+    for (const status of Object.keys(CLAIMS_STATUSES)) {
+      if (claimStatusesPresent.includes(status)) {
```

Comment:
The statuses come back to the frontend as `Title Case Like This`. Assuming the actual status casing matches the keys of the mapping table is going to cause problems, I bet.

Reply:
now that `vets-api` is acting as a more proper transform layer from the BTSSS API, I'm removing the mapping from the FE entirely, so this shouldn't be a concern now